### PR TITLE
Define api for validations in Admin

### PR DESCRIPTION
**Pull request by neilkidd**
**What**  
A simple free form field that should contain JSON

**Why**  
Admins can complete a tick list against each application

**Anything else?**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -274,9 +274,12 @@ definitions:
 
   ApplicationPatch:
     type: 'object'
+    description: 'One of status OR validations'
     properties:
       status:
         type: 'string'
+      validations:
+        type: 'string'
```
Comment (sirLisko) [inline swagger.yaml:282]: It's going to be an `object`, is it easier for you to have a `string`?

Not a big deal, I will just need to stringify and parse it.

Comment (neilkidd) [reply swagger.yaml:282]: If we have string, I can simply save and return. Happy to switch it though.
 WDYT?

File: swagger.yaml [hunk 2]
```diff
@@ -274,9 +274,12 @@ definitions:
 
   ApplicationPatch:
     type: 'object'
+    description: 'One of status OR validations'
     properties:
```
Comment (WillGibson) [inline swagger.yaml:278]: @neilkidd Did you decide against the `oneOf` thing?

File: swagger.yaml (hunk 1 above)
Comment (sirLisko) [reply swagger.yaml:282]: Ok then, let's leave the whole logic in the FE 👍

File: swagger.yaml (hunk 2 above)
Comment (neilkidd) [reply swagger.yaml:278]: It would be a slightly more significant change and is API 3. So for now I went for this

**Pull request merged**